### PR TITLE
[Merged by Bors] - feat(Topology): some trivial equivalences with the set of clopens

### PR DESCRIPTION
**Pull request by dagurtomas**
This is needed for the construction of limits and colimits in `LightProfinite`.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by dagurtomas** [comment]
The equivalence I really want is `LocallyConstant X (Fin 2) ≃ Clopens X` (see #10391), but I can't write that in either file, because neither imports the other, and I didn't want to change the imports in these files.

**Comment by adri326** [comment]
I don't see why you couldn't change the imports of `LocallyConstant/Basic.lean` to import `Sets/Closeds.lean` instead, as it didn't seem to cause any issues on my end

**Comment by dagurtomas** [comment]
Yeah it doesn't cause any issues, but I'm always wary of changing the import structure in these basic files. Maybe @semorrison can comment on whether it's ok to change the imports of `LocallyConstant/Basic.lean`?

**Comment by dagurtomas** [comment]
Maybe the least harmful option is to create a new, tiny file `LocallyConstant/Clopens.lean` which imports `LocallyConstant.Basic` and `Sets.Closed`, with only this equivalence?

**Comment by adri326** [comment]
That would be an alternative solution, yeah, if adding the import really doesn't work

**Comment by dagurtomas** [comment]
!bench

**Comment by dagurtomas** [comment]
!bench

**Comment by dagurtomas** [comment]
!bench

File: Mathlib/Topology/LocallyConstant/Basic.lean [hunk 1]
```diff
@@ -601,6 +601,18 @@ def congrLeft [TopologicalSpace Y] (e : X ≃ₜ Y) : LocallyConstant X Z ≃ Lo
...
 
+variable (X) in
+/--
+The set of clopen subsets of a topological space is equivalent to the locally constant maps to
+a two-element set
+-/
+def equivClopens [∀ (s : Set X) x, Decidable (x ∈ s)] :
+    LocallyConstant X (Fin 2) ≃ TopologicalSpace.Clopens X where
+  invFun s := ofIsClopen s.2
+  toFun f := ⟨f ⁻¹' {0}, f.2.isClopen_fiber _⟩
```
Comment (jcommelin) [inline Mathlib/Topology/LocallyConstant/Basic.lean:612]:
```suggestion
  toFun f := ⟨f ⁻¹' {0}, f.2.isClopen_fiber _⟩
  invFun s := ofIsClopen s.2
```

**Review by jcommelin — APPROVED** [review]
Usually we write `toFun` before `invFun` (-;

bors d+

**Comment by dagurtomas** [comment]
bors merge

**Pull request closed without merging**